server/testutils/factories/inmateDetail.ts

```diff
@@ -0,0 +1,15 @@
+import { faker } from '@faker-js/faker/locale/en_GB'
+import { Factory } from 'fishery'
+
+import FactoryHelpers from './factoryHelpers'
+import offenceHistoryDetail from './offenceHistoryDetail'
+import type { InmateDetail, OffenceHistoryDetail } from '@prison-api'
+
+export default Factory.define<Partial<InmateDetail>>(() => {
+  const offenceHistory = [offenceHistoryDetail.build({ mostSerious: true }), offenceHistoryDetail.build()]
+
+  return {
+    offenceHistory: FactoryHelpers.optionalArrayElement([offenceHistory]) as Array<OffenceHistoryDetail>,
```

Comment:
Turns out it isn't and must have been my VS code being a bit slow :D.  It was complaining about the type not matching and I assumed it was because of the use of Partial.  But removed it and all seems ok still.

Reply:
Ah, it happens!